### PR TITLE
Do not edit comment

### PR DESCRIPTION
**Pull request by zigomir**
Regarding #10.

We can get wording a bit better, I'm not a native speaker.

I also changed tests a bit so you can easily run them without needing to drop tables and types every time.

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -14,8 +14,16 @@ import * as diff from 'diff'
...
+        let formattedOutput = await typescriptOfSchema(
+            db,
+            'osm',
+            ['users'],
+            extractCommand(
+                ['node', 'schemats', 'generate', '-c', 'postgres://secretUser:secretPassword@localhost/test', '-t', 'users', '-o', 'osm.ts'],
+                'postgres://secretUser:secretPassword@localhost/test'
+            )
+        )
+        let outputFile = (process.env.CIRCLE_ARTIFACTS || '.') + '/osm.ts'
```
Comment (xiamx) [inline test/test.ts]: Please output the file `osm.ts` inside `test` directory. [Gitignore](https://github.com/SweetIQ/schemats/pull/11/files#diff-a084b794bc0759e7a6b77810e01874f2R342) can also be updated to reflect the change.

File: test/example/osm.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
 
+/**
+ * AUTO-GENERATED FILE - DO NOT EDIT!
+ *
+ * This file was generated with schemats node package:
+ * $ schemats generate -c postgres://username:password@localhost/test -t users -o osm.ts
+ *
+ * Re-run command above if your DB schema was changed.
```
Comment (xiamx) [inline test/example/osm.ts]: Re-run the command above

**Comment by vitaly-t** [comment]
If I may suggest, the header should be in its own file, with formatting parameters to be resolved by `pg-promise`, using `pgp.as.format`. It is the most logical way to do it.

One of the parameters that surely is needed - Data/Time of when the header was generated, without which the header is like half-useless.

**Comment by xiamx** [comment]
> If I may suggest, the header should be in its own file, with formatting parameters to be resolved by pg-promise, using pgp.as.format. It is the most logical way to do it.

I see, so a separate file for handling header related logics. I'm okay with how it's implemented currently, as the header is relatively lean, if we need to add more to the header we can refactor it in the future. What do you think @zigomir ?

> Data/Time of when the header was generated

Yes this will indeed be very helpful. I'm wondering if it's better to expose the datetime of generation as a constant (so it is possible to console.log it programmatically)

**Comment by zigomir** [comment]
I don't see a strong need to move header contents into a file.
I 100% agree regarding time. I'm not sure what do you mean by exposing date time as a constant? I'd just put it somewhere in comment section in YYYY-MM-DD hh:mm:ss format.

**Comment by xiamx** [comment]
I was thinking about having generated ts file extended with a datetime constant, for example

```typescript
export namespace somenamespace {
    interface someTable {
        ...
    }
    const _declarationGenerationDate = ${now} // can use a shorter and more concise constant name
}
```

But that might need some careful planning if users are to use these constant in their code.

For now I think YYYY-MM-DD hh:mm:ss datetime in the header comment works well.

**Comment by zigomir** [comment]
@xiamx done. Also I believe circle should pass again now.

**Comment by xiamx** [comment]
Thank you @zigomir

**Pull request merged**